### ISSUE TITLE
#debug() throws an error with no arguments given

### ISSUE THREAD

**Issue by queckezz** [staltz/xstream#87]
From the docs:

> If you don't provide the spy argument, then debug will just console.log each event.

I read this as, if you call `.debug()` it should just `console.log` the event. However this throws an error:

```
TypeError: s is not a function(…)
```

With a spy function or label it works as expected:

<sub>[JSFiddle to try out](https://jsfiddle.net/Lahf60yj/2/)</sub>
```js
xs.of(1).debug('test')
// -> test: 1
xs.of(1).debug(console.log.bind(console))
// -> 1
```

Is this intended behaviour? I thought the default was just logging to the console when no arguments are provided.

**Comment by staltz** [comment]
Oops, we have ourselves a bug there. Thanks for reporting

**Comment by staltz** [comment]
Fixed in v5.3.1
